onnxruntime/test/onnx/main.cc

```diff
@@ -348,24 +360,12 @@ int real_main(int argc, char* argv[], OrtEnv** p_env) {
 #endif
   // clang-format on
 
-#ifdef _WIN32
-  broken_tests["tf_inception_resnet_v2"] = "failed: type mismatch";
-  broken_tests["tf_inception_v4"] = "failed: type mismatch";
```

Comment:
I'm curious why they need so much memory. For example, inception_v4, its model weights are only about 160MB. But at run time, it needs about 1.6GB memory.  If I understand correctly, the objects inside ITestCase are every small, they should be only a few KBs. So, how does this change can help fix the issue?

Reply:
Good question - it is not about how much inception_v4 need - it is about when it asks for that 1.6 GB, how much memory is left in the process to be allocated.
Suppose before inception_v4, there are already N cases loaded up and tested, inception_v4 is more likely to failed for bad_allocation than the situation of 0 cases ( because those N cases are released instantly after test done).

Reply:
Doesn't mean if we run the models by one by, the problem will be gone?